packages/manager/apps/dedicated/package.json

```
@@ -29,6 +29,7 @@
     "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
     "@ovh-ux/manager-catalog-price": "^1.0.1",
     "@ovh-ux/manager-cda": "^0.0.0 || ^1.0.0",
+    "@ovh-ux/manager-cdn-dedicated": "^0.0.0",
```

```suggestion
    "@ovh-ux/manager-cdn-dedicated": "^0.0.0 || ^0.1.0-alpha", 
```
for Semvers to be compatible once released